src/test/java/com/artipie/npm/proxy/RxNpmProxyStorageTest.java

```diff
@@ -51,7 +55,20 @@
     /**
      * Last modified date for both package and asset.
      */
-    private static final String DEF_LAST_MODIFIED = "Tue, 24 Mar 2020 12:15:16 GMT";
+    private static final String LAST_MODIFIED = "Tue, 24 Mar 2020 12:15:16 GMT";
+
+    /**
+     * Last updated date for package (datetime).
+     */
+    private static final OffsetDateTime LAST_UPDATED = OffsetDateTime.of(
```

Comment:
@brastak rename this variable, its not obvious what's the difference between `last_modified`